sdk/src/azure/core/az_mqtt5_rpc_server.c

```diff
@@ -12,39 +12,23 @@ AZ_NODISCARD az_mqtt5_rpc_server_options az_mqtt5_rpc_server_options_default()
 {
   return (az_mqtt5_rpc_server_options){
     .subscribe_timeout_in_seconds = AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS,
+    .subscription_topic_format
```

Comment:
It feels like this mixes a sample with the actual SDK. Maybe the default formats should be AZ_SPAN_EMPTY?